app/javascript/components/supplemental_application/sections/RentalAssistance.js

```diff
@@ -137,23 +138,24 @@ const validateError = (values) => {
 }
 
 const AddRentalAssistanceForm = ({ values, onSave, loading, onClose, applicationMembers, onDelete, isNew }) => {
-  const applicationMembersOptions = applicationMembers.map(member => (
-    {
+  const applicationMembersOptions = applicationMembers.map(member => {
+    return {
       label: `${member.first_name} ${member.last_name}`,
       value: member.id
     }
-  ))
+  })
 
   return (
     <Form onSubmit={onSave} defaultValues={values} validateError={validateError}>
       {formApi => (
-        <div className='app-editable expand-wide scrollable-table-nested'>
+        <div className='app-editable expand-wide scrollable-table-nested rental-assistance-form'>
```

Comment:
This class and the others added below this in this file are added to enable us to test. The e2e tests need identifiers to be able to locate and interact with the rental assistance form components.